src/Windowing/Silk.NET.SDL/SdlContext.cs

```diff
@@ -102,7 +103,34 @@ public void Dispose()
         public nint GetProcAddress(string proc, int? slot = default)
         {
             AssertCreated();
-            return (nint) _sdl.GLGetProcAddress(proc);
+            _sdl.ClearError();
+            var ret = (nint) _sdl.GLGetProcAddress(proc);
+            _sdl.ThrowError();
+            if (ret == 0)
+            {
+                throw new SymbolLoadingException(proc);
+            }
+
+            return ret;
+        }
+
+        public bool TryGetProcAddress(string proc, out nint addr, int? slot = default)
+        {
+            addr = 0;
+            _sdl.ClearError();
+            if (_ctx is null)
+            {
+                return false;
+            }
```

Comment:
why do we care for _ctx here?

Reply:
We can't load OpenGL functions without a valid OpenGL context.

Reply:
While that makes sense logically, I don't get it from a code perspective. Seems like we could call into SDL just fine without OpenGL

Reply:
Possibly, but then we'll possibly do a completely unnecessary native call which isn't cheap, and for a Try method - a defensive method style - it makes sense to put in defensive logic in here.

Reply:
It's also an intentional design choice - we don't want any of the methods on the context object to be "alive" until the context that is represents is also "alive". Other methods do this using `AssertCreated` which throws if `ctx` is `null`, while this one eliminates the throw.